p2p/dialer.go

```diff
@@ -110,6 +114,15 @@ func (m *dialer) exchangeConfig(consumerID, providerID identity.Identity, servic
 	}
 	defer brokerConn.Close()
 
+	ready := make(chan bool)
+	defer close(ready)
+	_, err = brokerConn.Subscribe(channelHandlersReadySubject(providerID, serviceType), func(msg *nats_lib.Msg) {
+		if err := m.channelHandlersReady(msg, ready); err != nil {
+			log.Err(err).Msg("handlers ready handler setup failed")
```

Comment:
Something wrong with this message. I cannot parse it. 
```suggestion
			log.Err(err).Msg("handlers ready handler setup failed")
```